athina/steps/llm.py

```
@@ -203,11 +204,16 @@ def execute(self, input_data: dict, **kwargs) -> str:
             api_formatted_messages = [msg.to_api_format() for msg in messages]
 
             llm_service_response = self.llm_service.chat_completion(
-                api_formatted_messages,  # Use the formatted messages here
+                messages,
                 model=self.model,
                 **self.model_options.model_dump(),
                 **(self.tool_config.model_dump() if self.tool_config else {}),
                 **({"response_format": self.response_format}),
+                **(
+                    kwargs.get("search_domain_filter", None)
```

Ruff (0.8.0)

Use `kwargs.get("search_domain_filter")` instead of `kwargs.get("search_domain_filter", None)`

Replace `kwargs.get("search_domain_filter", None)` with `kwargs.get("search_domain_filter")`

(SIM910)